scripts/perf/delete_instance/delete_instance.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -euo pipefail
+
+VAULT_TOKEN=$(cat /etc/vault-token-dsde)
+
+DOCKER_ETC_PATH=/usr/share/etc
+
+mkdir -p mnt
+
+# Read the service account credentials from vault:
+docker run --rm -e VAULT_TOKEN=$VAULT_TOKEN broadinstitute/dsde-toolbox vault read -format=json secret/dsp/cromwell/perf/service-account-deployer | jq -r '.data.service_account' > mnt/sa.json
```

Comment:
`jq -r --exit-status '.data.service_account'`, or better yet DRY this file out by moving common functions into an include like `helper.inc.sh`.